### ISSUE TITLE
presentChoiceSet failed with "INVALID_ID, null"

### ISSUE THREAD

**Issue by yingchun1** [smartdevicelink/sdl_java_suite#1336]
### Bug Report
presentChoiceSet work well If just new ChoiceCell with text but will fail if i add the ChoiceCell with VrCommands in it.
##### Reproduction Steps
1. Follow the sample in sdl to add VR strings in ChoiceCell to support VR commands
2. run the code

##### Expected Behavior
it should show the choice set successful 

##### Observed Behavior
but it returns error with message "INVALID_ID, null"

##### OS & Version Information
* Android Version: android 8.0.0
* SDL Android Version: 4.10.1

##### Test Case, Sample Code, and / or Example App
```
ChoiceCell cell1 = new ChoiceCell("Item 1", Arrays.asList("Item 1"), null);
ArrayList choiceCellList = new ArrayList<>(Arrays.asList(cell1));
sdlManager.getScreenManager().preloadChoices(choiceCellList, success -> {if(success) {
    Logger.i(TAG, " preloadChoices success ");
    ChoiceSet choiceSet = new ChoiceSet("Choose an Item from the list", choiceCellList, new ChoiceSetSelectionListener() {
        @Override
        public void onChoiceSelected(ChoiceCell choiceCell, TriggerSource triggerSource, int rowIndex) {
            Logger.i(TAG, choiceCell.getText() + " was selected");
        }
        @Override
        public void onError(String error) {
            Logger.e(TAG, "There was an error showing the perform interaction: "+ error);
        }
    });
    sdlManager.getScreenManager().presentChoiceSet(choiceSet, InteractionMode.BOTH);
} else {
    Logger.i(TAG, " preloadChoices fail ");
}});
```

**Comment by bilal-alsharifi** [comment]
Hi @yingchun1 What head unit are you testing against. I tried your code on Manticore and it worked well.

**Comment by yingchun1** [comment]
i am using Ford TDK SYNC3, software version 2.3 build 17109_DEVTEST

**Comment by bilal-alsharifi** [comment]
> i am using Ford TDK SYNC3, software version 2.3 build 17109_DEVTEST

 @yingchun1 do you mean version 3.2? I thought SYNC3 versions will always be something like 3.X

**Comment by bilal-alsharifi** [comment]
I tried to run the provided code on a similar TDK which is Ford SYNC3 `v3.4 build 19353_DEVTEST_r133796` as I don't have access to a TDK with the exact same version that you provided. I haven't seen the issue. Also, wasn't able to see it on Manticore.
It could be an issue from that specific TDK. Would it be possible to reproduce the issue on Manticore?

**Comment by theresalech** [comment]
@yingchun1 could you please test PR #1405 with both Manticore and your TDK?  We believe #1405 should resolve the issue you've reported.  Thanks!

**Comment by theresalech** [comment]
Closing as we believe this issue to be resolved via #1405.  Please let us know if you are still experiencing the issue.  Thank you!
